### DISCUSSION TITLE
The arm platform cannot load fonts. I need your help

### DISCUSSION THREAD

**Discussion by asdone7** [ocornut/imgui#4777] [New Users Build/Link/Run issues ONLY!]
hello，

I can load fonts well under Ubuntu, but I can't load fonts when I use cross compilation to generate binary files for arm platform.

I added regular.ttf font in the way of "ImFont* font = io.Fonts->AddFontFromFileTTF". After the construction is completed, open the binary file, all the symbols displayed are "？"。
I checked the construction log and prompted me whether "void gui_init()': unused variable 'font'" is related to this. However, I also have this prompt when building under ubuntu, and it does not display abnormity

Thanks in advance.

**Comment by andreamancuso** [comment]
Hi, I am experiencing the same issue on my Raspberry Pi 5. I am using GLFW3 and OpenGL 3.

**Reply by andreamancuso** [reply]
I reduced the number of fonts drastically (I was loading quite a lot, including Font Awesome) and it is working fine now.
